packages/upgrades/src/interface.cairo

```diff
@@ -7,3 +7,11 @@ use starknet::ClassHash;
 pub trait IUpgradeable<TState> {
     fn upgrade(ref self: TState, new_class_hash: ClassHash);
 }
+
+#[starknet::interface]
+pub trait IUpgradeAndCall<TState> {
+    fn upgrade(ref self: TState, new_class_hash: ClassHash);
```

Comment:
This interface doesn't need both methods, but only the `upgrade_and_call` one, if for some reason someone wants both, they can implement both interfaces.